app/src/components/HomePage/HomePage.jsx

```diff
@@ -1,19 +1,28 @@
-import React from "react";
-import hyfLogo from "../../assets/hyf.svg";
-import "./HomePage.css";
+"use client";
```

Comment:
You don't need to use `'use client'` here since you're using Vite instead of Next.js